assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -33,13 +35,25 @@ function create(){
     /*eslint-enable */
     window.ga('create', guardian.googleAnalytics.trackingId, {
         'allowLinker': true,
-        'name': tracker,
+        'name': defaultTracker,
         'cookieDomain': guardian.googleAnalytics.cookieDomain
     });
-    return (a,b,c) => window.ga(tracker+ '.' + a,b,c);
+    return gaProxy
 }
 
-export function init() {
+export function gaProxy() {
+    if (gaEnabled) {
+        let allArgs = Array.from(arguments);
```

Comment:
just FYI really but you can still use `const` here even though you're mutating the array - arrays (and objects) are always mutable in JS, `const` just means that you can't reassign it, whereas you can with `let`.  it's always better to use `const` unless you're reassigning the variable later on.